tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -109,4 +126,68 @@ public function test_get_post_type_icon() {
 		$valid            = $this->call_private_method( $post_type_object, 'get_post_type_icon' );
 		$this->assertContains( 'data:image/svg+xml;base64', $valid );
 	}
+
+	/**
+	 * @covers \Google\Web_Stories\Story_Post_Type::admin_enqueue_scripts
+	 */
+	public function test_admin_enqueue_scripts() {
+		$post_type_object = new \Google\Web_Stories\Story_Post_Type();
+		set_current_screen( 'post.php' );
+		get_current_screen()->post_type = \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG;
+		get_current_screen()->base      = 'post';
+		$post_type_object->admin_enqueue_scripts( 'post.php' );
+		$this->assertTrue( wp_script_is( \Google\Web_Stories\Story_Post_Type::WEB_STORIES_SCRIPT_HANDLE, 'registered' ) );
+		$this->assertTrue( wp_style_is( \Google\Web_Stories\Story_Post_Type::WEB_STORIES_SCRIPT_HANDLE, 'registered' ) );
+	}
+
+	/**
+	 * @covers \Google\Web_Stories\Story_Post_Type::filter_site_kit_gtag_opt
+	 */
+	public function test_filter_site_kit_gtag_opt() {
+		global $wp_query;
+		$wp_query->is_singular    = true;
+		$wp_query->queried_object = get_post( self::$story_id );
```

Comment:
I think you can just do `$this->go_to( get_permalink( self::$story_id ) )` to set the current query correctly. That's what core does.

Reply:
Fixed